pkg/controller/ps/status.go

```diff
@@ -34,13 +34,15 @@ func (r *PerconaServerMySQLReconciler) reconcileCRStatus(ctx context.Context, cr
 	if cr == nil || cr.ObjectMeta.DeletionTimestamp != nil {
 		return nil
 	}
+	if err := r.Get(ctx, client.ObjectKeyFromObject(cr), cr); err != nil {
+		return errors.Wrap(err, "get cluster")
+	}
```

Comment:
this will overwrite anything we assign to status fields in Reconcile function, right?

Reply:
Yes, but it shouldn't cause any issues since we overwrite each field of the status in the `reconcileCRStatus` method

Reply:
i am not sure if it's okay, users test failure might be caused by this